sources/packages/web/src/views/student/StudentAppealRequest.vue

```diff
@@ -84,31 +87,32 @@ export default defineComponent({
   components: {
     AppealRequestsForm,
   },
-  setup() {
+  props: {
+    applicationId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props) {
     const snackBar = useSnackBar();
     const processing = ref(false);
     const appealRequestsForms = ref([] as StudentAppealRequest[]);
-    let applicationId: number;
+    const initialData = ref({} as StudentRequestSelectedForms);
+    let applicationAppealData: ApplicationProgramYearAPIOutDTO;
     const showRequestForAppeal = computed(
       () => appealRequestsForms.value.length === 0,
     );
 
-    const submitRequest = async (
-      form: FormIOForm<StudentRequestSelectedForms>,
-    ) => {
+    onMounted(async () => {
       try {
-        const application =
+        applicationAppealData =
           await ApplicationService.shared.getApplicationForRequestChange(
-            form.data.applicationNumber,
+            props.applicationId,
           );
-        applicationId = application.id;
-        appealRequestsForms.value = form.data.formNames.map(
-          (formName) =>
-            ({
-              formName,
-              data: { programYear: application.programYear },
-            } as StudentAppealRequest),
-        );
+        initialData.value = {
+          applicationNumber: applicationAppealData.applicationNumber,
+          formNames: [],
+        };
       } catch (error: unknown) {
         const errorMessage = "An error happened while requesting a change.";
         const errorLabel = "Unexpected error";
```

Comment:
May or may not be part of the PR. Other than the unexpected error, other warning snackbars aren't relevant anymore right?

Reply:
`INVALID_APPLICATION_NUMBER` is now removed.